### PR TITLE
Add exercise test GHA

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

**Comment by BNAndras** [comment]
Closes #101

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    
+        - name: Run Tests
+          run: |
+            box task run tasks/TestAllSolutions
+
+
+            
+
+
+
```
Comment (ErikSchierboom) [inline .github/workflows/test.yml:59]:
```suggestion
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    runs-on: ubuntu-22.04
+
+    steps:
+        - name: Checkout code
+          uses: actions/checkout@v2
```
Comment (ErikSchierboom) [inline .github/workflows/test.yml:30]: Could you pin this to a SHA?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    
+        - name: Set up Java
+          uses: actions/setup-java@v2
```
Comment (ErikSchierboom) [inline .github/workflows/test.yml:33]: Could you pin this to a SHA?

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+          with:
+            distribution: 'adopt'
+            java-version: '8'
+    
+        - name: Cache CommandBox
+          uses: actions/cache@v2
```
Comment (ErikSchierboom) [inline .github/workflows/test.yml:39]: Could you pin this to a SHA?

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**